src/lib.rs

```diff
@@ -19,9 +19,47 @@ pub use tables::GeneralCategory;
 /// that this version of unicode-general-category was generated from.
 pub const UNICODE_VERSION: (u64, u64, u64) = (16, 0, 0);
 
+impl GeneralCategory {
+    /// Returns the general category abbreviation.
+    pub fn as_str(&self) -> &'static str {
+        match self {
```

Comment:
Please add a link to where these values were retrieved from. This will make it easier for folks in the future if a new category is added and the abbreviated value needs to be added to this match.